### ISSUE TITLE
.vmf parser should ignore invalid values

### ISSUE THREAD

**Issue by Blakepane** [lasa01/io_import_vmf#83]
I'm trying to load the GMod map [rp_downtown_tits_v2](https://steamcommunity.com/sharedfiles/filedetails/?id=1590239460) into blender so I can make a render of the map for a "guide". 

I've gotten the workshop .gma file into a .bsp through the built in gmad program that installs with gmod. From there, I used bspscr to get it into the .vmf and the other folder (forgot what it's called). I followed the video tutorial, and I was able to get everything to work fine with a csgo map, but when I tried importing rp_downtown_tits_v2, I got an error while on [1/12] Parsing map. 

This is the full error.

```
Python: Traceback (most recent call last):
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 92, in _parse_float
    return float(value)
ValueError: could not convert string to float: 'yes'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\__init__.py", line 748, in execute
    importer.load(self.filepath, context, self.map_data_path)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\import_vmf.py", line 140, in load
    vmf = vmfpy.VMF(open(file_path, encoding="utf-8"), self._vmf_fs)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 771, in __init__
    entity_inst = self._parse_custom(VMFSpotLightEntity, "entity (light_spot)", entity, self.fs)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 138, in _parse_custom
    return parser(*args)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 400, in __init__
    super().__init__(data, fs)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 378, in __init__
    self.constant_attn = self._parse_float_str("_constant_attn", data)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 100, in _parse_float_str
    return self._parse_float(name, value)
  File "C:\Users\blake\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\io_import_vmf\deps\vmfpy\__init__.py", line 94, in _parse_float
    raise VMFParseException(f"{name} is not a valid float", self._context)
vmfpy.VMFParseException: VMF parsing failed: entity (light_spot) (id 103128) _constant_attn is not a valid float

location: <unknown location>:-1
```

I've loaded the game folders for all source games I have installed in case it's somehow needed, and there was still no luck.

If you need any more information about this please let me know.

**Comment by lasa01** [comment]
That light seems to have invalid properties in the map. Seems like in games they are ignored, but my vmf parser is more strict.

You can work around the issue by opening the .vmf file in notepad, searching for the id that's in the error message (103128) and deleting the line that specifies the `_constant_attn` that's invalid.

**Comment by lasa01** [comment]
This is fixed in Plumber 1.0.0 beta versions, you can try the latest version [here](https://github.com/lasa01/io_import_vmf/releases/tag/v1.0.0-beta3).
